test/commands/scorecard.test.js

```diff
@@ -97,27 +97,55 @@ test("should not display scorecard for unknown repository", async() => {
   assert.deepEqual(givenLines, expectedLines, `lines should be ${expectedLines}`);
 });
 
-test("should retrieve repository whithin git config", async() => {
-  const testingModule = await esmock("../../src/commands/scorecard.js", {
-    fs: {
-      readFileSync: () => [
-        "[remote \"origin\"]",
-        "\turl = git@github.com:myawesome/repository.git"
-      ].join("\n")
-    }
-  });
-
-  assert.deepEqual(testingModule.getCurrentRepository(), Ok(["myawesome/repository", "github"]));
+// test("should retrieve repository whithin git config", async() => {
+//   const testingModule = await esmock("../../src/commands/scorecard.js", {
+//     fs: {
+//       readFileSync: () => [
+//         "[remote \"origin\"]",
+//         "\turl = git@github.com:myawesome/repository.git"
+//       ].join("\n")
+//     }
+//   });
+
+//   assert.deepEqual(testingModule.getCurrentRepository(), Ok(["myawesome/repository", "github"]));
+// });
+test("should retrieve repository within git config", async () => {
+  const fs = await import('fs');
+  const readFileSyncMock = mock.method(fs, 'readFileSync', () =>
+    [
+      '[remote "origin"]',
+      '\turl = git@github.com:myawesome/repository.git',
+    ].join('\n')
+  );
+  const testingModule = await import("../../src/commands/scorecard.js");
+
+  assert.deepEqual(
+    testingModule.getCurrentRepository(),
+    Ok(["myawesome/repository", "github"])
+  );
```

Comment:
```suggestion
  const readFileSyncMock = mock.method(fs, 'readFileSync', () =>
    [
      '[remote "origin"]',
      '\turl = git@github.com:myawesome/repository.git',
    ].join('\n')
  );

  assert.deepEqual(
    testingModule.getCurrentRepository(),
    Ok(["myawesome/repository", "github"])
  );
```

You can add `import fs from "node:fs";` and `import * as testingModule from "../../src/commands/scorecard.js";` at the top of the file, we don't need dynamic imports
(same below + in `httpServer.test.js`)

Reply:
fixed in latest commit